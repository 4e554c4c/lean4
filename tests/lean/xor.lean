#eval (10 : USize).lxor (5 : USize)
#eval (10 : USize).lxor (6 : USize)
#eval (10 : Nat).lxor (5 : Nat)
#eval (10 : Nat).lxor (10 : Nat)
