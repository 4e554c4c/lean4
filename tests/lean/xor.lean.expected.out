15
12
15
0
